app/views/forms/_screen.html.erb

```diff
@@ -37,8 +37,101 @@
 
     </div>
 
-    <%# questions on the form %>
-    <%= index_table(Questioning, @form.questionings, :within_form => true) if form_mode != :new %>
+    <div class="questions">
+      <input class="btn btn-primary" type="submit" value="Save"></input>
```

Comment:
This button shouldn't be in this div I think. Should be above as it's part of the main form, not the question list.